tests/test_core.py

```diff
@@ -5981,6 +5981,11 @@ def test_sse_diagnostics(self):
       stderr=PIPE)
     self.assertContained('Instruction emulated via slow path.', p.stderr)
 
+  @requires_native_clang
+  def test_relaxed_simd_implies_simd128(self):
+    src = test_file('sse/test_sse1.cpp')
+    self.build(src, emcc_args=['-mrelaxed-simd', '-msse']);
```

Comment:
This will fix the flake8 bot:
```suggestion
    self.build(src, emcc_args=['-mrelaxed-simd', '-msse'])
```

Reply:
Also, see the definition of the `wasm_simd` decorator at the top of this file for the wasm2js fix. Specifically, the test should be skipped if `not self.is_wasm()`. It might be worth creating a new `wasm_relaxed_simd` decorator, actually.

Reply:
Good idea, done!